superx/information_extractors/branch_info_extractor.py

```diff
@@ -31,13 +39,13 @@ def run_branch_extractor(self):
                 if self.current_super['needs_web_scraping']:
                     zip_link = self.get_zip_file_link()
                     if self.current_super['need_zip_prefix']:
-                        self.current_super['branch_url'] = self.current_super['branch_url'] + zip_link
+                        self.current_super['branch_url'] = self.current_super['branch_url'] + zip_link #pylint: disable=line-too-long
```

Comment:
why are you disabling this check? Put the code on multiple lines

Reply:
done